doc/references.bib

```diff
@@ -106,6 +131,21 @@ @article{Bellec2010
 }
 
+@article{Chen2011,
+	title = {Cortical surface-based searchlight decoding},
+	journal = {NeuroImage},
+	volume = {56},
+	number = {2},
+	pages = {582-592},
+	year = {2011},
+	note = {Multivariate Decoding and Brain Reading},
+	issn = {1053-8119},
+	doi = {https://doi.org/10.1016/j.neuroimage.2010.07.035},
+	url = {https://www.sciencedirect.com/science/article/pii/S1053811910010086},
+	author = {Yi Chen and Praneeth Namburi and Lloyd T. Elliott and Jakob Heinzle and Chun Siong Soon and Michael W.L. Chee and John-Dylan Haynes},
+	abstract = {Local voxel patterns of fMRI signals contain specific information about cognitive processes ranging from basic sensory processing to high level decision making. These patterns can be detected using multivariate pattern classification, and localization of these patterns can be achieved with searchlight methods in which the information content of spherical sub-volumes of the fMRI signal is assessed. The only assumption made by this approach is that the patterns are spatially local. We present a cortical surface-based searchlight approach to pattern localization. Voxels are grouped according to distance along the cortical surface—the intrinsic metric of cortical anatomy—rather than Euclidean distance as in volumetric searchlights. Using a paradigm in which the category of visually presented objects is decoded, we compare the surface-based method to a standard volumetric searchlight technique. Group analyses of accuracy maps produced by both methods show similar distributions of informative regions. The surface-based method achieves a finer spatial specificity with comparable peak values of significance, while the volumetric method appears to be more sensitive to small informative regions and might also capture information not located directly within the gray matter. Furthermore, our findings show that a surface centered in the middle of the gray matter contains more information than to the white–gray boundary or the pial surface.}
```

Comment:
wonder if it makes sense to have the abstract in our bib file?

Reply:
Not really. But this is minor...